hunting-buddy.lic

```diff
@@ -76,6 +78,16 @@ class HuntingBuddy
     bput("stow my #{description}", 'You put')
   end
 
+  def check_prebuffs
+    return unless @settings@settings.pre_hunt_buff 
+    return unless !@settings.waggle_sets.nil?
+
+    if @settings.pre_buff_room
+      walk_to(@settings.pre_buff_room)
+    end
+    wait_for_script_to_complete('buff', ['prehunt'])
```

Comment:
Do you have a character using this waggle set? If so, would you be up for including their YAML in this PR so there is an example to work from?